thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/dashboard/resources/DetectionJobResource.java

```diff
@@ -589,6 +591,11 @@ public Response tuneAlertFilter(@PathParam("functionId") long id,
     BaseAlertFilter alertFilter = alertFilterFactory.fromSpec(anomalyFunctionSpec.getAlertFilter());
     // create alert filter auto tune
     AutotuneConfigDTO autotuneConfig = new AutotuneConfigDTO(alertFilter);
+    if (StringUtils.isNotBlank(features)) {
+      Properties autotuneProperties = autotuneConfig.getTuningProps();
+      autotuneProperties.setProperty(AUTOTUNE_FEATURE_KEY, features);
```

Comment:
Here you are changing feature set. Please log the previous feature set and your updated features.

Reply:
The old feature is now logged in the properties. Thanks for the comment